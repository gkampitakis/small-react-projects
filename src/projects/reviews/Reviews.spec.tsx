export {};

describe('Reviews', () => {
  it.todo('Should render correctly');

  it.todo('Should render next review');

  it.todo('Should render previous review');

  it.todo('Should circle through reviews');

  it.todo('Should show a random review');
});
