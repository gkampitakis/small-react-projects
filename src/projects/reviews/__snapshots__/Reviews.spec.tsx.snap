// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews Should circle through reviews 1`] = `
<body
  style="color: rgb(16, 16, 16); background: rgb(241, 241, 241);"
>
  <div>
    <main
      class="reviews_project"
    >
      <section
        class="container"
      >
        <div
          class="title"
        >
          <h2>
            Our Reviews
          </h2>
          <div
            class="underline"
          />
        </div>
        <article
          class="review-card"
        >
          <div
            class="img-container"
          >
            <img
              alt="susan smith's Avatar"
              src="https://res.cloudinary.com/diqqf3eq2/image/upload/v1586883334/person-1_rfzshl.jpg"
            />
            <span
              class="quote-icon"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </span>
          </div>
          <h4
            class="author"
          >
            susan smith
          </h4>
          <p
            class="job"
          >
            web developer
          </p>
          <p
            class="info"
          >
            I'm baby meggings twee health goth +1. Bicycle rights tumeric chartreuse before they sold out chambray pop-up. Shaman humblebrag pickled coloring book salvia hoodie, cold-pressed four dollar toast everyday carry
          </p>
          <div
            class="btn-container"
          >
            <button
              class="prev-btn"
              data-testid="prev-btn"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 320 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
                />
              </svg>
            </button>
            <button
              class="next-btn"
              data-testid="next-btn"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 320 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
                />
              </svg>
            </button>
          </div>
          <button
            class="random-btn"
          >
            surprise me
          </button>
        </article>
      </section>
    </main>
  </div>
</body>
`;

exports[`Reviews Should render correctly 1`] = `
<main
  className="reviews_project"
>
  <section
    className="container"
  >
    <div
      className="title"
    >
      <h2>
        Our Reviews
      </h2>
      <div
        className="underline"
      />
    </div>
    <article
      className="review-card"
    >
      <div
        className="img-container"
      >
        <img
          alt="susan smith's Avatar"
          src="https://res.cloudinary.com/diqqf3eq2/image/upload/v1586883334/person-1_rfzshl.jpg"
        />
        <span
          className="quote-icon"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
            />
          </svg>
        </span>
      </div>
      <h4
        className="author"
      >
        susan smith
      </h4>
      <p
        className="job"
      >
        web developer
      </p>
      <p
        className="info"
      >
        I'm baby meggings twee health goth +1. Bicycle rights tumeric chartreuse before they sold out chambray pop-up. Shaman humblebrag pickled coloring book salvia hoodie, cold-pressed four dollar toast everyday carry
      </p>
      <div
        className="btn-container"
      >
        <button
          className="prev-btn"
          data-testid="prev-btn"
          onClick={[Function]}
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 320 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
            />
          </svg>
        </button>
        <button
          className="next-btn"
          data-testid="next-btn"
          onClick={[Function]}
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 320 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
            />
          </svg>
        </button>
      </div>
      <button
        className="random-btn"
        onClick={[Function]}
      >
        surprise me
      </button>
    </article>
  </section>
</main>
`;

exports[`Reviews Should render next review 1`] = `
<body
  style="background: rgb(241, 241, 241); color: rgb(16, 16, 16);"
>
  <div>
    <main
      class="reviews_project"
    >
      <section
        class="container"
      >
        <div
          class="title"
        >
          <h2>
            Our Reviews
          </h2>
          <div
            class="underline"
          />
        </div>
        <article
          class="review-card"
        >
          <div
            class="img-container"
          >
            <img
              alt="anna johnson's Avatar"
              src="https://res.cloudinary.com/diqqf3eq2/image/upload/v1586883409/person-2_np9x5l.jpg"
            />
            <span
              class="quote-icon"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </span>
          </div>
          <h4
            class="author"
          >
            anna johnson
          </h4>
          <p
            class="job"
          >
            web designer
          </p>
          <p
            class="info"
          >
            Helvetica artisan kinfolk thundercats lumbersexual blue bottle. Disrupt glossier gastropub deep v vice franzen hell of brooklyn twee enamel pin fashion axe.photo booth jean shorts artisan narwhal.
          </p>
          <div
            class="btn-container"
          >
            <button
              class="prev-btn"
              data-testid="prev-btn"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 320 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
                />
              </svg>
            </button>
            <button
              class="next-btn"
              data-testid="next-btn"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 320 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
                />
              </svg>
            </button>
          </div>
          <button
            class="random-btn"
          >
            surprise me
          </button>
        </article>
      </section>
    </main>
  </div>
</body>
`;

exports[`Reviews Should render previous review 1`] = `
<body
  style="color: rgb(16, 16, 16); background: rgb(241, 241, 241);"
>
  <div>
    <main
      class="reviews_project"
    >
      <section
        class="container"
      >
        <div
          class="title"
        >
          <h2>
            Our Reviews
          </h2>
          <div
            class="underline"
          />
        </div>
        <article
          class="review-card"
        >
          <div
            class="img-container"
          >
            <img
              alt="bill anderson's Avatar"
              src="https://res.cloudinary.com/diqqf3eq2/image/upload/v1586883423/person-4_t9nxjt.jpg"
            />
            <span
              class="quote-icon"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M464 32H336c-26.5 0-48 21.5-48 48v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48zm-288 0H48C21.5 32 0 53.5 0 80v128c0 26.5 21.5 48 48 48h80v64c0 35.3-28.7 64-64 64h-8c-13.3 0-24 10.7-24 24v48c0 13.3 10.7 24 24 24h8c88.4 0 160-71.6 160-160V80c0-26.5-21.5-48-48-48z"
                />
              </svg>
            </span>
          </div>
          <h4
            class="author"
          >
            bill anderson
          </h4>
          <p
            class="job"
          >
            the boss
          </p>
          <p
            class="info"
          >
            Edison bulb put a bird on it humblebrag, marfa pok pok heirloom fashion axe cray stumptown venmo actually seitan. VHS farm-to-table schlitz, edison bulb pop-up 3 wolf moon tote bag street art shabby chic. 
          </p>
          <div
            class="btn-container"
          >
            <button
              class="prev-btn"
              data-testid="prev-btn"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 320 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
                />
              </svg>
            </button>
            <button
              class="next-btn"
              data-testid="next-btn"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 320 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
                />
              </svg>
            </button>
          </div>
          <button
            class="random-btn"
          >
            surprise me
          </button>
        </article>
      </section>
    </main>
  </div>
</body>
`;
